packages/compass-indexes/src/components/indexes-toolbar/indexes-toolbar.tsx

```diff
@@ -44,7 +44,7 @@ export const IndexesToolbar: React.FunctionComponent<IndexesToolbarProps> = ({
   const showCreateIndexButton = !isReadonly && !isReadonlyView && !errorMessage;
 
   return (
-    <Toolbar className={toolbarStyles}>
+    <Toolbar className={toolbarStyles} data-testid="indexes-toolbar">
```

Comment:
How do you feel about updating the `toolbarStyles` here to make the toolbar transparent/white?
I'm thinking it looks a bit nicer and is closer to where the designs are going. Not quite related to this work, but I think updating this table makes the default toolbar background stick out a bit now.

Before:
<img width="962" alt="Screen Shot 2022-08-19 at 2 41 13 PM" src="https://user-images.githubusercontent.com/1791149/185686016-1615b501-51a8-4a9e-8ca6-123056b755e5.png">

After:
<img width="962" alt="Screen Shot 2022-08-19 at 2 41 32 PM" src="https://user-images.githubusercontent.com/1791149/185686044-b0d35fba-1763-4e05-8763-e87f0059c7ed.png">

Reply:
Actually wanted to focus on toolbar in the next PR where we will add `Refresh Indexes` button. But as @alenakhineika  also added a comment somewhat related, i fixed it.